hack/update-schemas.sh

```diff
@@ -30,3 +36,8 @@ SCHEMAPATCH_CONFIG_FILE="$(dirname $0)/schemapatch-config.yaml" controller-gen \
   schemapatch:manifests=config/core/300-resources,generateEmbeddedObjectMeta=true \
   output:dir=config/core/300-resources \
   paths=./pkg/apis/...
+
+# Restore linked CRDs.
+for link in $links; do
+  mv "$link.bkp" "$(readlink -f "$link")"
```

Comment:
think Im just misunderstanding some horribly bash thing, but just to put me out of my misery 😅.. why isn't this just `mv "$link.bkp" "$link"`?

Reply:
because it's a link. I tried that before and it overrode the link to be just the file for me :joy:

Reply:
hm, looks like `readlink` doesnt have `-f` on my mac :(. Do I have to install some dependency?

Reply:
(I get `readlink: illegal option -- f` when I try doing this)

Reply:
BSD stuff... should be fixed now.